src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/prisonerprofile/PrisonerProfileService.kt

```diff
@@ -16,6 +16,7 @@ class PrisonerProfileService(
   fun getPhysicalAttributes(offenderNo: String) =
     repository.findAllByOffenderNomsId(offenderNo)
       .takeIf { it.isNotEmpty() }
+      ?.filterNot { it.physicalAttributes.isEmpty() }
```

Comment:
tiny unrelated - but might be worth renaming `repository` to `bookingRepository` - this confused the hell out of me thinking it was finding all prisoners with the offenderNo